testproject/django_file_form_example/views.py

```diff
@@ -15,6 +16,11 @@ class BaseFormView(generic.FormView):
     use_ajax = True
     custom_js_file = 'example_form.js'
 
+    def get_initial(self):
```

Comment:
I modified this example for convenience. We should remove this and add a separate example for placeholder.